serverless.yml

```diff
@@ -26,8 +25,13 @@ provider:
     ProvisionedBy: Serverless
     Project: projectworks-leave-notifications
 package:
-  exclude:
-    - .env
+  patterns:
```

Comment:
I'd recommend using the [`patterns` from our template](https://github.com/ackama/serverless-aws-template/blob/main/serverless.yml#L56-L84); you'll just need to replace the last line with `src/**` and `handler.js`

Reply:
Good idea. Will do.